core/__tests__/models/destination/plugins/exportRecord.ts

```diff
@@ -955,6 +955,72 @@ describe("models/destination - with custom exportRecord plugin", () => {
       });
     });
 
+    describe("when destination delivery mode is", () => {
+      let record: GrouparooRecord = null;
+
+      beforeEach(async () => {
+        record = await helper.factories.record();
+        await record.addOrUpdateProperties({
+          email: ["newEmail@example.com"],
+        });
+
+        await destination.exportRecord(record, true);
+      });
+
+      afterEach(async () => {
+        await record.destroy();
+      });
```

Comment:
You may want to `Export.truncate()` between tests as well, to ensure that there is no test polution